### ISSUE TITLE
Steam effecs

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#2119]
Do you have any plans for a return of the steam emitting Panel C? It really did add a wonderful touch to some mechanisms and made the old  PR materI am much more lively. I'm looking to rebuild some of the deploying guns/turrets that were used in the Kickstarter campaign and a steam/sparks emitting brick would be an awesome call back to the old days
https://imgur.com/vVPgFJU

**Comment by ZachZent** [comment]
Other effects would be nice as well. Steam, sparks, maybe even arcs of energy. Always made for pretty things

**Comment by Kaiser-Indrasil** [comment]
Lightning arcs would be awesome. Even in a form of 2 separate blocks with empty space between them and lightning showing up every so often. With logic support perhaps?

**Comment by Womble-TC** [comment]
Could even make it into a feature, have some kind of block that dissipates large amounts heat at the cost for some power, have it have a cloud of steam as it does so. (Having it as a decoration would be nice also too though)

**Comment by Servomotor** [comment]
Looking to purchase game tonight. Been watching for what seems like forever. I love the steam emitter/effect when doors are opened. Other effects would also be greatly welcomed. Anything that can add that little bit of immersion and "life" to the builds.

**Comment by tsunamayo** [comment]
Yes I will try to have them back before ea! For electrical ark we will see, I had other plan with that.

**Comment by ProPeach** [comment]
Yahoo! Awesome :smiley:

**Comment by Servomotor** [comment]
Woot! One of my favorite bits of the promotional material...the steam!!!! STEAM!!!

**Comment by Kaiser-Indrasil** [comment]
Steam, before Steam release ;)
